.buildkite/scripts/build_and_upload_packages.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
```

Comment:
I'd name this script something like `build_and_upload_firecracker_packages.sh`.

Reply:
done